src/ec/packages/ec-component-timeline/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@ecl-twig/ec-component-timeline",
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "2.7.0",
+  "description": "ECL Twig - EC Timeline",
+  "dependencies": {
+    "@ecl-twig/ec-component-button": "2.3.2",
+    "@ecl-twig/ec-component-icon": "2.3.2"
```

Comment:
Now you don't need `"@ecl-twig/ec-component-icon": "2.3.2"` anymore since `@ecl-twig/ec-component-button` takes care of adding the icon